pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -266,6 +266,17 @@ func (c *InstallerController) newNodeStateForInstallInProgress(currNodeState *op
 	failed := false
 	errors := []string{}
 
+	// Store status of this deployment in the status configmap for revision history tracking
+	statusConfigMap, err := c.kubeClient.CoreV1().ConfigMaps(c.targetNamespace).Get(getStatusConfigMapName(currNodeState.TargetRevision, currNodeState.NodeName), metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	statusConfigMap.Data[strconv.Itoa(int(currNodeState.TargetRevision))] = string(installerPod.Status.Phase)
```

Comment:
> Data["nodeName.phase"]

Actually, just `Data["phase"]`

Reply:
@deads2k I think we should clarify the structure of the configmap we're using. Are you saying that each revision's configmap should only hold the phase for that revision? It seems like that would take a lot more work to build the list of successful/failed revisions if we can only get one data point from each revision's configmap.

What I'm trying to do is have each revision's configmap hold every other revision's statuses. The data would be copied over and updated by the InstallerController each time a new revision is made, and the PruneController can also clean up the configmap so it doesn't get massive. Then it only takes one api call to get the info (but we still need to sort it)

I don't know how we could use one data point per configmap without going back and loading in each revision's configmap data (from file?) If this isn't actually that expensive of an operation I'm fine with it, and it would let us sort the statuses in-place as we read them. I just want to make sure I understand your approach. Thoughts?